.github/workflows/scripts/install-build-deps.sh

```diff
@@ -22,6 +22,10 @@ do
   fi
 done
 
+echo -U "$USER_AGENT" --retry-connrefused --waitretry=5 --read-timeout=20 --timeout=15 -t 5 https://downloadmirror.intel.com/"$SDE_MIRROR_ID"/"$SDE_TAR_NAME"-lin.tar.xz
```

Comment:
You need to defined `USER_AGENT` in the build job to make it work. See here:
https://github.com/ispc/ispc/blob/8ec6a69016fc10f62827fe0b275005a07f1d1ed0/.github/workflows/ispc-ci.yml#L27

Reply:
Right!